--- /opt/dlami/nvme/patch/orig/kimi_k3.py	2026-09-01 01:29:49.000000000 +0000
+++ /opt/dlami/nvme/patch/kimi_k3.py	2026-09-01 09:03:25.400397498 +0000
@@ -542,6 +542,7 @@
         self._ep_a2a = (
             _a2a_backend.is_megamoe()
             or _a2a_backend.is_deepep()
+            or _a2a_backend.is_deepep_v2()
             or _a2a_backend.is_mooncake()
             or _a2a_backend.is_ascend_fuseep()
             or _a2a_backend.is_mori()
@@ -2192,6 +2193,7 @@
             (
                 _a2a_backend.is_megamoe()
                 or _a2a_backend.is_deepep()
+                or _a2a_backend.is_deepep_v2()
                 or _a2a_backend.is_mooncake()
                 or _a2a_backend.is_ascend_fuseep()
                 or _a2a_backend.is_mori()
